### ISSUE TITLE
"--showdiffs" is ignored?

### ISSUE THREAD

**Issue by simonbcn** [Jguer/yay#597]
#### Affected Version
yay v8.922

#### Issue
Your manpage says:
```
...
PERMANENT CONFIGURATION SETTINGS
...
--showdiffs
Show diffs for build files. Diffs are shown via git diff which uses
less by default. This behaviour can be changed via git's config,
the $GIT_PAGER or $PAGER environment variables.
...
```
but:
```
$ yay --save --showdiffs           
$ grep diff .config/yay/config.json      
	"answerdiff": "",
	"diffmenu": true,
```

**Comment by Morganamilo** [comment]
That was changed in 8430c41b. Looks like it was never taken out of the man page.

**Comment by Morganamilo** [comment]
Fixed in #594
